.gitignore

```diff
@@ -17,4 +17,7 @@ api/types/graphql.d.ts
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+
+# For XState typegen
+web/**/*.typegen.ts
```

Comment:
Typegen resources here:
- https://stately.ai/blog/introducing-typescript-typegen-for-xstate
- https://xstate.js.org/docs/guides/typescript.html#typegen